lib/loggly.js

```diff
@@ -0,0 +1,16 @@
+import winston from 'winston';
+import 'winston-loggly';
+
+export function addTransport(winstonLogger) {
```

Comment:
I think we should start best practice of putting all export keywords at the end of the file rather than inline with the function. Might need discussion. Doesn't make much difference here, but it's easier if the rule is always the same.